### PR TITLE
Abacus 2.0: Result message isn’t shown after performing Delete Entitled Attribute operation.

### PR DESCRIPTION
**Pull request by ysizarie-virtru**
### Changelog:
- add toast messages on delete action in entitlements/clients;
- update gitignore;

### CODE REVIEW

File: src/containers/User/User.tsx [hunk 1]
```diff
@@ -83,10 +83,14 @@ const User = () => {
...
-                .delete(`/entitlements/${entity.attribute}`, {
-                    data: [entity.entityId],
-                })
-                .then(() => getEntitlements(config));
+              .delete(`/entitlements/${entity.entityId}`, {
+                data: [entity.attribute],
+              })
+              .then(() => {
+                getEntitlements(config);
+                toast.success(`Entity ${entity.entityId} deleted`);
```
Comment (sievdokymov-virtru) [inline src/containers/User/User.tsx:91]: @ysizarie-virtru We're deleting attribute entitlement, not entity. Could you pls update param?

File: src/containers/Client/Client.tsx [hunk 2]
```diff
@@ -44,10 +45,14 @@ const Client = () => {
...
       entitlementsClient
-        .delete(`/entitlements/${entity.attribute}`, {
-          data: [entity.entityId],
+        .delete(`/entitlements/${entity.entityId}`, {
+          data: [entity.attribute],
         })
-        .then(() => getEntitlements(config));
+        .then(() => {
+          getEntitlements(config);
+          toast.success(`Entity ${entity.entityId} deleted`);
```
Comment (sievdokymov-virtru) [inline src/containers/Client/Client.tsx:53]: @ysizarie-virtru Same here with inconsistent parameter.

File: src/containers/User/User.tsx (hunk 1 above)
Comment (ysizarie-virtru) [reply src/containers/User/User.tsx:91]: @sievdokymov-virtru  Like **_'Attribute <name> deleted'_** ?

Comment (sievdokymov-virtru) [reply src/containers/User/User.tsx:91]: Yep, seems suitable

**Comment by ysizarie-virtru** [comment]
@sievdokymov-virtru  please test this branch locally before merging

**Review by sievdokymov-virtru — APPROVED** [review]
Good to go.

**Pull request merged**